### PR TITLE
created test rmds to aid YJ rule development

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

**Comment by yunjoonjung-PNNL** [comment]
Thank you! I'll make use of these TCDs and if I have any questions, I'll get back to this PR to ask questions.

After using these TCDs, I'll approve this PR.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                                                            "id": "Supply Fan 1"
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1"
+                                                        }
+                                                    ],
+                                                    "air_energy_recovery": {
+                                                        "energy_recovery_operation": "WHEN_MINIMUM_OUTSIDE_AIR"
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json:72]: Could you add ```id``` (something like ```"id": "Air Energy Recovery 1",```)? The ```id``` key is a required key and without it the TCD is not valid against the schema.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+                                                        {
+                                                            "id": "Supply Fan 1"
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1"
+                                                        }
+                                                    ],
+                                                    "air_energy_recovery": {
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json:185]: Could you also add id (something like "id": "Air Energy Recovery 1",) here?

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+                                                        "heating_source": "ELECTRIC",
+                                                        "fan": {
+                                                            "id": "Terminal Fan 1"
+                                                        },
+                                                        "supply_design_heating_setpoint_temperature": 105
+                                                    }
+                                                ],
+                                                "design_thermostat_heating_setpoint": 65,
+                                                "spaces": [
+                                                    {
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:51]: Could you add ```id``` here?

**Comment by yunjoonjung-PNNL** [comment]
I tried to test out Rule 19-15 and found a few things in the TCD.

1. ```id``` is missing under the ```spaces``` keys.
2.  ```id``` is missing under the ```fan_system``` keys.
3.  ```id``` is missing under the ```supply_fans``` keys.

You can also validate the TCD (instance) against the schema using the ```run_json_schema_validation.py``` developed by Juan. Please let me know if you have any questions!

**Comment by JacksonJ-KC** [comment]
> I tried to test out Rule 19-15 and found a few things in the TCD.
> 
> 1. `id` is missing under the `spaces` keys.
> 2. `id` is missing under the `fan_system` keys.
> 3. `id` is missing under the `supply_fans` keys.
> 
> You can also validate the TCD (instance) against the schema using the `run_json_schema_validation.py` developed by Juan. Please let me know if you have any questions!

Thanks for the reminder! It had been a couple of months since I last worked on TCDs. I will validate moving forward.
Added ids to 'spaces'. Removed references to 'fan_system' and 'supply_fans' and replaced with terminals[0].fan parameters, as this is testing terminal systems.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+            "ruleset_reference": "G3.1.2.8.2",
+            "rule_description": "For baseline system types 9 & 10, the system design supply airflow rates  shall be based on the temperature difference between a supply air temperature set point of 105\u00b0F and the design space-heating temperature set point, the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater.",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:19]: Could we have the ```proposed```?

Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:73]: Added

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+                                                        "type": "CONSTANT_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 10",
+                                                        "heating_source": "ELECTRIC"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
+                                                "id": "System 10",
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:49]: Could you double-check this system? When I used the ```get_baseline_system_types``` function, it says "Error: HVAC system 10 does not match any baseline system types.".

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+                                                    "id": "Furnace Coil 1",
+                                                    "heating_system_type": "FURNACE",
+                                                    "energy_source_type": "ELECTRICITY"
+                                                },
+                                                "fan_system": {
+                                                    "id": "CAV Fan System 1",
+                                                    "fan_control": "CONSTANT",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1"
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:60]: Could you add the ```is_airflow_automated``` key? This is required in the ```are_all_hvac_sys_fan_objs_autosized```.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json (hunk 5 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:49]: should be resolved now, let me know if not

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json (hunk 6 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:60]: added

**Pull request closed without merging**